### PR TITLE
Fixing pkg bld error due to expired certs

### PR DESCRIPTION
**Pull request by parameshbabu**
Specifying the new 2017 cert explicitly in the parameter and also fixing a minor issue in the script to use the settings.

### CODE REVIEW

File: TurnkeySecurity/IotDeviceGuardBitLocker.psm1 [hunk 1]
```diff
@@ -25,15 +25,16 @@ function GenerateBitlockerDataRecoveryAgentPackageXml([xml] $config)
         $blob = (Get-ItemProperty HKLM:\SOFTWARE\Microsoft\SystemCertificates\My\Certificates\$thumbprint).Blob
 
         # convert 'blob' from array to 'int' to a comma delimited string of hex values
```
Comment (bfjelds) [inline TurnkeySecurity/IotDeviceGuardBitLocker.psm1]: is this comment appropriate anymore?  says a comma delimited string, but the join isn't using a comma anymore?

File: TurnkeySecurity/IotDeviceGuardBitLocker.psm1 [hunk 2]
```diff
@@ -25,15 +25,16 @@ function GenerateBitlockerDataRecoveryAgentPackageXml([xml] $config)
...
-        $blob = (($blob | foreach {$_.ToString("x2") }) -join ',')
+        #$blob = (($blob | foreach {$_.ToString("x2") }) -join ',')
```
Comment (bfjelds) [inline TurnkeySecurity/IotDeviceGuardBitLocker.psm1]: remove commented code

Comment (parameshbabu) [reply TurnkeySecurity/IotDeviceGuardBitLocker.psm1]: Done.

File: TurnkeySecurity/IotDeviceGuardBitLocker.psm1 (hunk 1 above)
Comment (parameshbabu) [reply TurnkeySecurity/IotDeviceGuardBitLocker.psm1]: Updated

**Pull request merged**